help_test.go

```diff
@@ -574,58 +574,6 @@ func TestShowCommandHelp_CommandAliases(t *testing.T) {
 	}
 }
 
-func TestHelpNameConsistency(t *testing.T) {
```

Comment:
what happened to this ?

Reply:
It's no longer necessary or valid given that `NewApp` is removed

Reply:
No no no. This is to test whether the help occurs consistently between

```
app help cmd
app cmd help
app cmd -h
app cmd subcmd help
```

Reply:
@dearchap https://github.com/urfave/cli/pull/1626/files#r1051682870